app/models/cinstance.rb

```diff
@@ -36,7 +36,7 @@ class Cinstance < Contract
 
   # this has to be before the include Backend::ModelExtensions::Cinstance
   # or callbacks order makes keys not to be saved in backend
-  after_save :create_first_key, on: :create
+  after_create :create_first_key
```

Comment:
same here

Reply:
I think here it makes sense to change to `after_create`, although not sure if that would be breaking something in the existing behavior :thinking:

Reply:
If I understood this correctly, `after_save` ignores the `on:` option in rails 5. So right now in master `create_first_key` is being called every time the application is `#updated`. `after_create` seems the correct behavior for me.

Reply:
I'm leaving it as `after_create`, I think it's better.